app/views/api/policies/edit.html.slim

```diff
@@ -1,7 +1,20 @@
 - content_for :page_header_title, 'Policies'
 
-= semantic_form_for @proxy, url: admin_service_policies_path(@service) do |f|
-  = render '/api/integrations/apicast/shared/policies', f: f
-  = f.actions do
-    = f.commit_button t('formtastic.actions.policies.update'), button_html: {class: 'update', name: 'update', value: '1',
-      id: 'policies-button-sav', title: 'Update Policies', disabled: (@error ? true : nil) }
+- unless @error.nil?
```

Comment:
Maybe `if @error.present?` for better readability?

Reply:
Dariaclimate 🥁